src/PowerShellEditorServices/Workspace/Workspace.cs

```diff
@@ -91,8 +91,9 @@ public ScriptFile CreateScriptFileFromFileBuffer(string filePath, string initial
         }
 
         /// <summary>
-        /// Gets an open file in the workspace.  If the file isn't open but
-        /// exists on the filesystem, load and return it.
+        /// Gets an open file in the workspace. If the file isn't open but exists on the filesystem, load and return it.
+        /// <para>IMPORTANT: Not all documents have a backing file e.g. untitled: scheme documents.  Consider using
```

Comment:
```suggestion
        /// <param>IMPORTANT: Not all documents have a backing file e.g. untitled: scheme documents.  Consider using
```

did you mean param here?